context/app/static/js/components/home/Hero/HeroTab.tsx

```diff
@@ -1,23 +1,39 @@
+import React, { ComponentType, ReactElement } from 'react';
+import { useEventCallback } from '@mui/material/utils';
 import Chip from '@mui/material/Chip';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
-import { InternalLink } from 'js/shared-styles/Links';
-import React, { ComponentType, ReactElement } from 'react';
 import Hidden from '@mui/material/Hidden';
+import { trackEvent } from 'js/helpers/trackers';
+import { InternalLink } from 'js/shared-styles/Links';
 import { HeroTabContainer } from './styles';
 import { useHeroTabContext } from './HeroTabsContext';
 
 interface HeroTabActionProps {
   title: string;
   icon: ReactElement;
+  tabTitle: string;
   onClick?: () => void;
   href?: string;
 }
 
-function HeroTabAction({ title, icon, onClick, href }: HeroTabActionProps) {
+function HeroTabAction({ title, icon, tabTitle, onClick, href }: HeroTabActionProps) {
+  const handleClick = useEventCallback(() => {
+    trackEvent({
+      category: 'Homepage',
+      action: `Hero / ${tabTitle}`,
+      label: `${title} Button`,
+    });
+
+    if (onClick) {
+      onClick();
+    }
```

Comment:
Alternatively:
```suggestion
    onClick?.();
```
But this is optional.

In other circumstances, I would also consider adjusting `handleClick` and the type definition for `onClick` so that the underlying event still gets forwarded; since `onClick` was being passed directly to the chip before, there is an implicit click event arg that still gets passed. In some circumstances the parent component may want to use this event directly. Since the onClick was already defined as an argless void in this case it's probably fine, though.